packages/snap/src/General.service.ts

```diff
@@ -57,10 +67,21 @@ class GeneralService {
    *
    * @returns void
    */
-  static async removeFriendlyDapp(dapp: string): Promise<void> {
+  static async removeFriendlyDapp(args: { dApp: string }): Promise<void> {
+    const content = panel([
```

Comment:
Maybe it would make sense to move all of the `const content = ...` into `UI.service.ts` and create methods for each confirmation:
- confirmRemoveFriendlyDapp
- confirmAddFriendlyDapp
- confirmSwitchMethods
....

I'm not completely sure about this, so its fine if we leave it like this :+1: